pvlib/iam.py

```diff
@@ -292,7 +292,6 @@ def martin_ruiz(aoi, a_r=0.16):
     pvlib.iam.interp
     pvlib.iam.sapm
     '''
-    # Contributed by Anton Driesse (@adriesse), PV Performance Labs. July, 2019
```

Comment:
I disagree that this was a copy/paste error; it was included in the original PR contributing this function: #752

Reply:
I don't know how that line was inserted at the head of the `martin_ruiz` function, as that wasn't contributed by Anton, but may have been edited to apply some processing that was copied from `martin_ruiz_diffuse` or other functions that Anton contributed.

Reply:
Perhaps I am misunderstanding something, but (ignoring irrelevant shuffling between modules) `martin_ruiz` was originally contributed in #752, which was indeed from Anton.

Reply:
@kandersolar is correct, I didn't follow the blame history to the origin.

The comment should be restored.